server/templates/home/organization.html

```diff
@@ -0,0 +1,34 @@
+{% extends "base.html" %}
+{% load static %}
+
+{% block head %}
+<link rel="stylesheet" href="{% static "css/organization.css" %}">
+{% endblock head %}
+
+{% block content %}
+{% comment %} <span id="organization">Organizations</span> {% endcomment %}
+<h3 id="org-tittle">Explore by organizations</h3>
+
+<div class='card-container'>
+{% for value in details %}
+<div class="card col-sm col-md-6 col-lg-3 ftco-animate">
+    <div class="organization">
+        <a href="#" class="img-prod"><img class="img-fluid" src="{{ value.file.url }}" alt="natural farming"></a>
```

Comment:
Why is this a link?

Reply:
When the user clicks the organization image, they will be redirected to the organization detail page